$NetBSD: patch-ae,v 1.6 2015/12/29 23:34:52 dholland Exp $

Instead of trying to probe whether unaligned accesses work, wire it
down safely.

--- config/auto-aux/align.c.orig	Sat Sep 27 19:00:30 2003
+++ config/auto-aux/align.c	Sat Sep 27 19:53:24 2003
@@ -98,6 +98,14 @@
   long n[1001];
   int speed_aligned, speed_unaligned;
 
+#if 1
+  /* ISO C is very restrictive about unaligned access, and absence of
+   * SIGBUS or SIGSEGV for these tests does not mean that the compiler
+   * will generate the expected code...  So report that unaligned acces
+   * does not work. */
+  exit(1);
+#endif
+
   if (test(access16, (char *) n + 1)) exit(1);
   if (test(access32, (char *) n + 1)) exit(1);
   if (test(access32, (char *) n + 2)) exit(1);
